index.js

```diff
@@ -28,10 +28,6 @@ exports.transform = exports.filter = function transform(config) {
   var handlers = {};
 
   if (db.type() === 'http') {
-    handlers.put = function (orig, args) {
-      args.doc = incoming(args.doc);
-      return orig();
-    };
```

Comment:
this  is no longer necessary in the latest pouchdb implementation; in fact otherwise it encrypts twice